assets/src/edit-story/components/library/panes/media/mediaPane.js

```diff
@@ -104,6 +104,19 @@ const Inner = styled.div`
     / 1fr;
 `;
 
+const Loading = styled.div`
+  position: absolute;
+  bottom: 20px;
+  left: 50%;
+  transform: translateX(-50%);
+  padding: 8px 80px;
+  background-color: rgba(0, 0, 0, 0.4);
+  border-radius: 100px;
+  font-size: 15px;
```

Comment:
Perhaps add font size & line height & font weight to the theme or re-use `theme.fonts.label` like above?

e.g. `font-size: ${({ theme }) => theme.fonts.label.size};`